app/views/layouts/application.html.erb

```diff
@@ -5,8 +5,10 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
+        
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous">
```

Comment:
I think it might be better to add bootstrap into the codebase like here https://medium.com/@frouster/bootstrap-with-ruby-on-rails-the-beginning-59c494d7a4f9 I guess if the "jsdelivr" CDN goes down we wont have bootstrap in our project. Best if we serve html and css from "our CDN".

Reply:
This didn't work for me, and I didn't have any `javascripts` directory under `app` as this article suggests

Reply:
I have now installed bootstrap in my latest commit